apm-agent-plugins/apm-ecs-logging-plugin/src/test/java/co/elastic/logging/jul/JulMdcAccessor.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package co.elastic.logging.jul;
+
+import java.util.Map;
+
+/**
+ * Test-only accessor for {@link JulMdc#getEntries()}
+ */
+public class JulMdcAccessor {
+
+    public static Map<String, String> getEntries(){
+        return JulMdc.getEntries();
```

Comment:
[for reviewer] the `getEntries` method being package-protected, this allows to provide access for tests.